routes/request.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+
+const Request = require('../models/Request')
+
+// GET route
+router.get("/", async (req, res) => {
+    const requests = await Request.find().sort("title");
+    res.send(requests);
+  });
+
+
+// Post Route
+  router.post('/', async (req, res) => {
+    const data = new Request({
+        title: req.body.title,
+        description: req.body.description,
+        isClaimed: req.body.isClaimed
```

Comment:
These are a good start to the request model data.
One question I have and there may be another way that you are planning to do this with the db modelling.
How are the requests linked to the users? Are the id's going to be listed in an array in the user schema?
Just wondering whether it is worth linking the userId to the request as well (which may not be necessary)

Reply:
So that will be a 'ref' to the User and yes, I think it would be worth linking the userId to the request. I'll add both to the trello board. Thanks for pointing that out!